manifests/agent/annotation.pp

```diff
@@ -0,0 +1,25 @@
+# @summary Add agent annotation
+#
+# @example
+#   sensu::agent::annotation { 'cpu.message'': value => 'foo' }
```

Comment:
change the example so you do not have foo

Reply:
Used fatigue check example.